pageserver/src/tenant/timeline.rs

```diff
@@ -1199,6 +1202,35 @@ impl Timeline {
             .unwrap_or(self.conf.default_tenant_conf.eviction_policy)
     }
 
+    fn get_evictions_low_residence_duration_metric_threshold(
+        tenant_conf: &TenantConfOpt,
+        default_tenant_conf: &TenantConf,
+    ) -> Duration {
+        tenant_conf
+            .evictions_low_residence_duration_metric_threshold
+            .unwrap_or(default_tenant_conf.evictions_low_residence_duration_metric_threshold)
+    }
+
+    pub(super) fn tenant_conf_updated(&self) {
+        // NB: Most tenant conf optiosn are read by background loops, so,
```

Comment:
```suggestion
        // NB: Most tenant conf options are read by background loops, so,
```

Reply:
Seems like it, or, I resolved but never committed the fixes.